agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -276,6 +276,16 @@ bool backhaul_manager::socket_disconnected(Socket *sd)
                 finalize_slaves_connect_state(true, slaves_sockets.front());
             }
             LOG(INFO) << "disconnected slave sockets has been deleted";
+
```

Comment:
Commit message wrong grammar - "he will be notified" -> "it will be notified"

Reply:
done